README.md

```diff
@@ -74,7 +41,7 @@ The current workflows exist in this repo with the following statuses:
 
 ## Tests
 
-Given that this repo is still new, there are no tests. When we add tests, we will update the README.
+Every task in this repository is covered by at least one test (see all of our tests in `tests/tools/`). These are run using [pytest-workflow](https://pytest-workflow.readthedocs.io/en/stable/).
```

Comment:
We probably should, but that would also need environment bootstrapping instructions. Probably want to include miniwdl config instructions bc miniwdl is hardcoded. IDK. Sounds like a hassle to fully document.

Do you think we can get away with just the following?
```
Tests can be run with the following command: `python -m pytest-workflow --git-aware`
```
And _*only*_ that? Can we let the user figure out the environment on their own? It's not _that_ complicated? (Yes I am just being lazy on this PR 😅 )

Reply:
I think that's fine for now. We should probably have a dev environment definition eventually. Since we're using Docker containers to do the actual runs, it should be relatively simple.